makelib/k8s_tools.mk

```diff
@@ -25,15 +25,15 @@ ISTIO_DOWNLOAD_TUPLE := osx
 endif
 
 # the version of kind to use
-KIND_VERSION ?= v0.9.0
+KIND_VERSION ?= v0.11.0
 KIND := $(TOOLS_HOST_DIR)/kind-$(KIND_VERSION)
 
 # the version of kubectl to use
-KUBECTL_VERSION ?= v1.17.11
+KUBECTL_VERSION ?= v1.21.1
 KUBECTL := $(TOOLS_HOST_DIR)/kubectl-$(KUBECTL_VERSION)
 
 # the version of kustomize to use
-KUSTOMIZE_VERSION ?= v3.3.0
+KUSTOMIZE_VERSION ?= v4.1.3
```

Comment:
Any breaking changes in this major version bump?

Reply:
Given how kustomize has broken behavior between versions, this is something I was worried about.  So far I have not come across an issue.

 I can test more targets and a few other projects and report back.

Reply:
From a Crossplane perspective we shouldn't be impacted by any changes :+1: